tests/test_client_functional.py

```diff
@@ -1,5 +1,3 @@
-"""HTTP client functional tests against aiohttp.web server"""
```

Comment:
Please keep the module docstring.

Reply:
Why? What is the point?

Reply:
When I open a python module, regardless of whether it is a test suite or not it helps me to understand what I'm about to see next, get some context and decide whether I need to read further.

Reply:
I have a different position: docstrings for public API only.
It should be just a comment otherwise.